src/components/DateInput.jsx

```diff
@@ -181,6 +176,8 @@ class DateInput extends React.Component {
           {...css(
             styles.DateInput_input,
             readOnly && styles.DateInput_input__readOnly,
+            focused && styles.DateInput_input__focused,
+            disabled && styles.DateInput_input__disabled,
           )}
           aria-label={placeholder}
```

Comment:
probably for another PR, but this shouldn't be set to `placeholder`. If a developer wanted to pass in whatever they set for the `placeholder` as both the label and the placeholder text (which I'd argue that they shouldn't), that should be an explicit decision and not a default one.

Reply:
Yeah, let's do that in a follow-up! :)